packages/@vue/cli/lib/util/ProjectPackageManager.js

```diff
@@ -88,8 +89,17 @@ class PackageManager {
     if (forcePackageManager) {
```

Comment:
How do we force the package manager?

Reply:
The `-m / --packageManager` option of `vue create`. Maybe we should add this option for `vue add` and `vue invoke` too?